### PR TITLE
1276 mwcore navigation

### LINKED ISSUE #1276: MWCore Navigation
**Describe the Issue**

- [ ] Switch from bottom navigation bar to navigation drawer
- [ ] In the navigation drawer, Add 'Tracing' as an option
- [ ] In the navigation drawer, Add 'Appointments' as an option
- [ ] In the navigation drawer, Add 'Reports' as an option

### PR DESCRIPTION
**Pull request by AbdulWahabMemon**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #1276 

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/engine/src/main/java/org/smartregister/fhircore/engine/appfeature/AppFeature.kt [hunk 1]
```diff
@@ -26,4 +26,6 @@ sealed class AppFeature(val name: String) {
   object PatientManagement : AppFeature(name = "PatientManagement")
   object HouseholdManagement : AppFeature(name = "HouseholdManagement")
   object DeviceToDeviceSync : AppFeature(name = "DeviceToDeviceSync")
+  object Hiv : AppFeature(name = "Hiv")
+  object TracingAndAppointment : AppFeature(name = "TracingAndAppointment")
```
Comment (pld) [inline android/engine/src/main/java/org/smartregister/fhircore/engine/appfeature/AppFeature.kt:30]: being nitpicky again here :D I think we should also rename this but I'm not sure what to yet...

Comment (AbdulWahabMemon) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/appfeature/AppFeature.kt]: good catch @pld , its was cloned and changes might got reverted, pushing with all patients as expected by default.

Comment (AbdulWahabMemon) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/appfeature/AppFeature.kt]: pushed

**Comment by AbdulWahabMemon** [comment]
> > should we change `TracingAndAppointment` to Appointment` throughout? I'm not sure if we should be we changed the feature to name
> 
> @AbdulWahabMemon thoughts on this?

changes pushed

**Review by ellykits — CHANGES_REQUESTED** [review]

**Review by pld — DISMISSED** [review]

**Comment by ellykits** [comment]
@AbdulWahabMemon Kindly delete the **mwcore configs** from the `resources` package. We are already have the configs in this repository [fhir-resources](https://github.com/opensrp/fhir-resources/tree/main/). We have also migrated the common configs to the assets directory update any of your tests that were using configs from the resources directory.

**Comment by KhumboLihonga** [comment]
Hey guys, I was trying out the PR and it looks great. I had a few comments:

- Change the nav bar title from "Malawi Core" to "Tingathe".
- In the nav bar, the feature titled "HIV" should be labelled "Client List" and so should the toolbar title of it's activity.
- In the nav bar, change "Appointment" to "Appointments".
- In the nav bar, remove "Tracing" and instead add, "Phone Tracing" as one feature and "Home Tracing" as another feature.
- Change the "Phone Tracing" icon to the call [icon](https://fonts.gstatic.com/s/i/short-term/release/materialsymbolsoutlined/call/default/48px.svg)
- Change the "Home Tracing" icon to the house [icon](https://fonts.gstatic.com/s/i/short-term/release/materialsymbolsoutlined/house/default/48px.svg)
- Change the "Appointments" icon to the event [icon](https://fonts.gstatic.com/s/i/short-term/release/materialsymbolsoutlined/event/default/48px.svg)

**Comment by ellykits** [comment]
Thanks @KhumboLihonga for the feedback @AbdulWahabMemon I think you can address most of the comments with configs with few updates in the code.

**Review by ellykits — APPROVED** [review]

**Pull request merged**